### ISSUE TITLE
Color Contrast for Link Color

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#309]
The best way to put us within the acceptable range is to:

- PatternFly-black-900 (default text color) from 292e34 to 282D33. This puts us at the 3:1 requirement for contrast between link and body text colors.
- Make all links or blue text semibold (that is body size)
- Change all the links/buttons/blue elements over the themed black areas to use pf-blue-300 to ensure enough contrast.

Prior github description, @jgiardino writes: 
_Currently, we're off by .04 in meeting the [color contrast requirements for link color](https://www.w3.org/TR/WCAG20-TECHS/G183.html) (when using color alone to indicate links). We can either change the color slightly so that we meet this requirement, or include some additional visual indication in addition to color._

**Comment by kybaker** [comment]
@jgiardino Can we adjust PatternFly-black-900 from 292e34 to 282D33? This puts us at the 3:1 requirement.

**Comment by jgiardino** [comment]
There's a similar issue that I noted in the [Vertical Nav PR](https://github.com/patternfly/patternfly-next/pull/311#pullrequestreview-127675189) where the link text and the light gray background lack sufficient contrast.

**Comment by kybaker** [comment]
After consultation with CSS and design the best way to put us within the acceptable range is to:
- PatternFly-black-900 (default text color) from 292e34 to 282D33. This puts us at the 3:1 requirement for contrast between link and body text colors.
- Make all links or blue text semibold (that is body size)
- Change all the links/buttons/blue elements over the themed black areas to use pf-blue-300 to ensure enough contrast.

**Comment by LHinson** [comment]
@mindreeper2420 any chance you could also help out with this one?
